web/js/doc.js

```diff
@@ -350,14 +350,14 @@ window.onscroll = (event) => {
     const accordionOptions = {
       collapsible: true,
       heightStyle: 'content',
-      beforeActivate: (event, ui) => {
-        position.scrollLeft = 0;
-        position.scrollTop = 0;
-
+      beforeActivate: function loadArticle(event, ui) {
```

Comment:
This seems unusual.  I'm surprised it's not a syntax error.  Any reason you're adding a name?

Reply:
> I'm surprised it's not a syntax error

It was an arrow function before, and that typically means that an old-fashioned counterpart could be also used, if needed.

I only wanted to be more descriptive of what's happening inside this function. In general, I tend to assign meaningful names to functions as it helps me with debugging and makes code more readable.

Reply:
> to avoid polluting the scope

What do you mean by that, exactly?

Reply:
I mean that named functions get dropped into the function-level scope, similar to `var` declarations.  That's another name the developer has to remember when they are reading the code, in case it gets called later.  When you write an arrow function, it's clear that no one is going to call it by its name.

Reply:
Got it, thanks for clarifying!